### PR TITLE
Tools library config version

### PR DESCRIPTION
**Pull request by heatheramz**
#### Description of changes: 
Address library versions in the Tools menu for the new IA - includes:
1- Create link to previous version for JS family users - https://tools-library-config-version.d9kfkg9g23gld.amplifyapp.com/react/tools/libraries/configure-categories/
2 - Update tools, library, and configure-categories for prev/tools section with navigation options since cannot use left menu - this included brining in the lib-v1 library page and adding an index page for tools so it is not empty.
- Examples:
-- https://tools-library-config-version.d9kfkg9g23gld.amplifyapp.com/react/prev/tools/
-- https://tools-library-config-version.d9kfkg9g23gld.amplifyapp.com/react/prev/tools/libraries/
-- https://tools-library-config-version.d9kfkg9g23gld.amplifyapp.com/react/prev/tools/libraries/configure-categories/

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [X] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [X] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/[platform]/prev/tools/index.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      meta
+    }
+  };
+}
+
+<InlineFilter filters={["javascript", "react-native", "angular", "nextjs", "react", "vue"]}>
+
+<Callout info>
+
+If you are working with an older project you can review information about the [pervious version of the Amplify libraries](/[platform]/prev/tools/libraries/) for your chosen framework or language. You can also review our guide to [configure Amplify categories for the previous version](/[platform]/prev/tools/libraries/configure-categories/).
```
Comment (hibler13) [inline src/pages/[platform]/prev/tools/index.mdx:38]: Insert comma after "older project" in the first line.

File: src/pages/[platform]/prev/tools/index.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  };
+}
+
+<InlineFilter filters={["javascript", "react-native", "angular", "nextjs", "react", "vue"]}>
+
+<Callout info>
+
+If you are working with an older project you can review information about the [pervious version of the Amplify libraries](/[platform]/prev/tools/libraries/) for your chosen framework or language. You can also review our guide to [configure Amplify categories for the previous version](/[platform]/prev/tools/libraries/configure-categories/). 
+
+For any new projects we recommend that you review the Amplify libraries content for the [latest version](/[platform]/tools/libraries/) for your chosen framework or language.
```
Comment (hibler13) [inline src/pages/[platform]/prev/tools/index.mdx:40]: Insert comma after "new projects" in the first line.

File: src/pages/[platform]/prev/tools/index.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+</Callout>
+
+</InlineFilter>
+
+<InlineFilter filters={["swift", "flutter", "android"]}>
+
+<Callout info>
+
+If you are working with an older project you can review information about the [pervious version of the Amplify libraries](/[platform]/prev/tools/libraries/) for your chosen framework or language.
```
Comment (hibler13) [inline src/pages/[platform]/prev/tools/index.mdx:50]: Insert comma after "older project" in the first line and correct typo in second line - "pervious"

File: src/pages/[platform]/prev/tools/index.mdx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+</InlineFilter>
+
+<InlineFilter filters={["swift", "flutter", "android"]}>
+
+<Callout info>
+
+If you are working with an older project you can review information about the [pervious version of the Amplify libraries](/[platform]/prev/tools/libraries/) for your chosen framework or language. 
+
+For any new projects we recommend that you review the Amplify libraries content for the [latest version](/[platform]/tools/libraries/).
```
Comment (hibler13) [inline src/pages/[platform]/prev/tools/index.mdx:52]: Insert comma after "new projects" in first line

File: src/pages/[platform]/prev/tools/libraries/index.mdx [hunk 5]
```diff
@@ -0,0 +1,78 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Amplify Libraries',
+  description:
+    'The Amplify open-source client libraries provide use-case centric, opinionated, declarative, and easy-to-use interfaces across different categories of cloud powered operations enabling mobile and web developers to easily interact with their backends.',
```
Comment (hibler13) [inline src/pages/[platform]/prev/tools/libraries/index.mdx:6]: Insert comma after "cloud powered operations"

File: src/pages/[platform]/prev/tools/libraries/index.mdx [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+import android_maintenance from '/src/fragments/lib-v1/android-maintenance.mdx';
+
+<Fragments fragments={{ android: android_maintenance }} />
+
+import flutter_maintenance from '/src/fragments/lib-v1/flutter-maintenance.mdx';
+
+<Fragments fragments={{ flutter: flutter_maintenance }} />
+
+The Amplify open-source client libraries provide use-case centric, and easy-to-use interfaces across different categories of cloud powered operations enabling mobile and web developers to easily interact with their backends. These libraries are powered by the AWS cloud. The libraries can be used with both new backends created using the Amplify CLI and existing backend resources.
```
Comment (hibler13) [inline src/pages/[platform]/prev/tools/libraries/index.mdx:46]: Insert comma after "cloud powered operations" and cap "cloud" in "AWS Cloud" (per AWS Style Guide).

File: src/pages/[platform]/prev/tools/libraries/index.mdx [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+import flutter2 from '/src/fragments/lib-v1/flutter.mdx';
+
+<Fragments fragments={{ flutter: flutter2 }} />
+
+<InlineFilter filters={["javascript", "react-native", "angular", "nextjs", "react", "vue"]}>
+
+<Callout info>
+
+If you are beginning a new project we recommend you work with the [latest version of the Amplify libraries](/[platform]/tools/libraries/) for your chosen framework or language.
```
Comment (hibler13) [inline src/pages/[platform]/prev/tools/libraries/index.mdx:64]: Insert comma after "new project" in the first line.

File: src/pages/[platform]/prev/tools/libraries/index.mdx [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+</Callout>
+
+## Amplify JavaScript
+
+The Amplify JavaScript libraries are supported for different web and mobile frameworks including React, React Native, Angular, and Vue.
```
Comment (hibler13) [inline src/pages/[platform]/prev/tools/libraries/index.mdx:70]: Insert comma after "mobile frameworks"

File: src/pages/[platform]/tools/libraries/configure-categories/index.mdx [hunk 9]
```diff
@@ -19,11 +19,15 @@ export function getStaticProps(context) {
...
-<InlineFilter filters={['javascript', 'react-native', 'angular', 'nextjs', 'vue','react']}>
-
 When using the Amplify CLI, the **amplifyconfiguration.json** file gets created and updated automatically for you based upon the resources you have added and configured.
 
-If you are not using the Amplify CLI or need to override these settings, this documentation shows the available configuration properties for each category.
+If you are not using the Amplify CLI or need to override these settings, this documentation shows the available configuration properties for each category. 
+
+<Callout info>
+
+If you are working with an older project you can also review the configuration properties for the [previous version](/[platform]/prev/tools/libraries/configure-categories/) for your chosen framework or language.
```
Comment (hibler13) [inline src/pages/[platform]/tools/libraries/configure-categories/index.mdx:28]: Insert comma after "older project" in the first line.

**Review by hibler13 — COMMENTED** [review]
@heatheramz Minor editorial changes buy otherwise this looks good.

File: src/pages/[platform]/prev/tools/index.mdx (hunk 1 above)
Comment (heatheramz) [inline src/pages/[platform]/prev/tools/index.mdx:38]:
```suggestion
If you are working with an older project, you can review information about the [pervious version of the Amplify libraries](/[platform]/prev/tools/libraries/) for your chosen framework or language. You can also review our guide to [configure Amplify categories for the previous version](/[platform]/prev/tools/libraries/configure-categories/). 
```

File: src/pages/[platform]/prev/tools/index.mdx (hunk 2 above)
Comment (heatheramz) [inline src/pages/[platform]/prev/tools/index.mdx:40]:
```suggestion
For any new projects, we recommend that you review the Amplify libraries content for the [latest version](/[platform]/tools/libraries/) for your chosen framework or language.
```

File: src/pages/[platform]/prev/tools/index.mdx (hunk 3 above)
Comment (heatheramz) [inline src/pages/[platform]/prev/tools/index.mdx:50]:
```suggestion
If you are working with an older project, you can review information about the [pervious version of the Amplify libraries](/[platform]/prev/tools/libraries/) for your chosen framework or language. 
```

File: src/pages/[platform]/prev/tools/index.mdx (hunk 4 above)
Comment (heatheramz) [inline src/pages/[platform]/prev/tools/index.mdx:52]:
```suggestion
For any new projects, we recommend that you review the Amplify libraries content for the [latest version](/[platform]/tools/libraries/).
```

File: src/pages/[platform]/prev/tools/libraries/index.mdx (hunk 5 above)
Comment (heatheramz) [inline src/pages/[platform]/prev/tools/libraries/index.mdx:6]:
```suggestion
    'The Amplify open-source client libraries provide use-case centric, opinionated, declarative, and easy-to-use interfaces across different categories of cloud powered operations, enabling mobile and web developers to easily interact with their backends.',
```

File: src/pages/[platform]/prev/tools/libraries/index.mdx (hunk 6 above)
Comment (heatheramz) [inline src/pages/[platform]/prev/tools/libraries/index.mdx:46]:
```suggestion
The Amplify open-source client libraries provide use-case centric, and easy-to-use interfaces across different categories of cloud powered operations, enabling mobile and web developers to easily interact with their backends. These libraries are powered by the AWS Cloud. The libraries can be used with both new backends created using the Amplify CLI and existing backend resources.
```

File: src/pages/[platform]/prev/tools/libraries/index.mdx (hunk 7 above)
Comment (heatheramz) [inline src/pages/[platform]/prev/tools/libraries/index.mdx:64]:
```suggestion
If you are beginning a new project, we recommend you work with the [latest version of the Amplify libraries](/[platform]/tools/libraries/) for your chosen framework or language.
```

File: src/pages/[platform]/prev/tools/libraries/index.mdx (hunk 8 above)
Comment (heatheramz) [inline src/pages/[platform]/prev/tools/libraries/index.mdx:70]:
```suggestion
The Amplify JavaScript libraries are supported for different web and mobile frameworks, including React, React Native, Angular, and Vue.
```

File: src/pages/[platform]/tools/libraries/configure-categories/index.mdx (hunk 9 above)
Comment (heatheramz) [inline src/pages/[platform]/tools/libraries/configure-categories/index.mdx:28]:
```suggestion
If you are working with an older project, you can also review the configuration properties for the [previous version](/[platform]/prev/tools/libraries/configure-categories/) for your chosen framework or language.
```

**Review by heatheramz — COMMENTED** [review]
Adding suggested changes to PR

File: src/pages/[platform]/prev/tools/index.mdx (hunk 1 above)
Comment (heatheramz) [inline src/pages/[platform]/prev/tools/index.mdx]:
```suggestion
If you are working with an older project you can review information about the [previous version of the Amplify libraries](/[platform]/prev/tools/libraries/) for your chosen framework or language. You can also review our guide to [configure Amplify categories for the previous version](/[platform]/prev/tools/libraries/configure-categories/). 
```

File: src/pages/[platform]/prev/tools/index.mdx (hunk 3 above)
Comment (heatheramz) [inline src/pages/[platform]/prev/tools/index.mdx]:
```suggestion
If you are working with an older project you can review information about the [previous version of the Amplify libraries](/[platform]/prev/tools/libraries/) for your chosen framework or language. 
```

**Review by hibler13 — APPROVED** [review]
LGTM

**Review by jacoblogan — APPROVED** [review]

**Pull request merged**